src/js/plugins/dom-chef@5.1.0/index.ts

```diff
@@ -0,0 +1,141 @@
+import { svgTagNames } from 'svg-tag-names';
```

Comment:
After looking at more of the code, isn't this available as a npm package?
So we can let npm handle the package for using https://www.npmjs.com/package/dom-chef